Headers/DebugServer2/Utils/CompilerSupport.h

```diff
@@ -66,23 +41,19 @@
 
 #if defined(COMPILER_MSVC)
 #define DS2_ATTRIBUTE_PACKED "DS2_ATTRIBUTE_PACKED not implemented on MSVC"
-#elif __has_attribute(__packed__)
+#elif defined(COMPILER_GCC) || defined(COMPILER_CLANG)
 #define DS2_ATTRIBUTE_PACKED __attribute__((__packed__))
-#else
-#error "Compiler attribute __packed__ is required."
 #endif
 
 #if defined(COMPILER_MSVC)
 #define DS2_ATTRIBUTE_ALIGNED "DS2_ATTRIBUTE_ALIGNED not implemented on MSVC"
-#elif __has_attribute(__aligned__)
+#elif defined(COMPILER_GCC) || defined(COMPILER_CLANG)
 #define DS2_ATTRIBUTE_ALIGNED(SIZE) __attribute__((__aligned__(SIZE)))
-#else
-#error "Compiler attribute __aligned__ is required."
```

Comment:
Might be worth keeping some of these `#errors` around, at least to tell the user that their compiler isn't supported.

Reply:
If the compiler doesn't support it, you'll get errors during the build anyway.

Reply:
True.